test/ipc/send.js

```diff
@@ -133,7 +133,7 @@ test('Does not hold message events on I/O errors', async t => {
 	const {exitCode, isTerminated, message, ipcOutput} = await t.throwsAsync(subprocess);
 	t.is(exitCode, 1);
 	t.false(isTerminated);
-	t.true(message.includes('Error: getOneMessage() cannot be used: the parent process has already exited or disconnected'));
+	t.true(message.includes('Error: getOneMessage()'));
```

Comment:
Those tests rely on a race condition between the time it takes to spawn a process vs to send an IPC message. Those fixes help prevent that.